deno.json

```diff
@@ -1,5 +1,6 @@
 {
   "tasks": {
+    "check": "typos . .github .vscode && deno fmt --check && deno lint && deno check src/main.tsx test/converter.test.ts",
```

Comment:
Using https://github.com/kachick/depop/blob/07d087eaa8795ad289ba90d8f55d85d3c5ebe095/deno.jsonc#L5C15-L5C134 globs made `Globs are currently not supported, but will be soon.`

I will update deno with the nix path